easybuild/easyconfigs/a/Arriba/Arriba-2.1.0-GCC-10.3.0.eb

```diff
@@ -0,0 +1,41 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+
+easyblock = 'MakeCp'
+
+name = 'Arriba'
+version = '2.1.0'
+
+homepage = 'https://github.com/suhrig/arriba'
+description = """Arriba is a command-line tool for the detection of gene fusions from RNA-Seq data.
+ It was developed for the use in a clinical research setting. Therefore, short runtimes and high
+ sensitivity were important design criteria."""
+
+toolchain = {'name': 'GCC', 'version': '10.3.0'}
+toolchainopts = {'pic': True}
+
+github_account = 'suhrig'
+source_urls = ['https://github.com/%(github_account)s/%(namelower)s/releases/download/v%(version)s']
+sources = ['%(namelower)s_v%(version)s.tar.gz']
+checksums = ['21c46121e00cf84ad471b2b22fca722d37e55bbdf9e7118978dc250754a264bb']
+
+dependencies = [
+    ('bzip2', '1.0.8'),
+    ('STAR', '2.7.9a'),
+    ('zlib', '1.2.11'),
+    ('XZ', '5.2.5'),
+]
+
+postinstallcmds = ['mkdir %(installdir)s/database']
+files_to_copy = ['%(namelower)s', 'documentation', 'download_references.sh', 'draw_fusions.R', 'LICENSE',
```

Comment:
@SethosII bit of a nitpick, but could you make this first line shorter please?

Reply:
@verdurin No problem, I changed it.